middleware.go

```diff
@@ -79,8 +79,6 @@ func parseRequestHeader(c *Client, r *Request) error {
 
 	if IsStringEmpty(hdr.Get(hdrUserAgentKey)) {
 		hdr.Set(hdrUserAgentKey, fmt.Sprintf(hdrUserAgentValue, Version))
-	} else {
-		hdr.Set("X-"+hdrUserAgentKey, fmt.Sprintf(hdrUserAgentValue, Version))
```

Comment:
I dont want to have that part included in my UA and i would leave that option to the user to include it. If you want to give the user the option, feel free to add a config flag to enable the appending to the UA, I wont and I cant since i am not a go programmer.